app/connectors/connectors/jms/activemq-connectors/activemq-model/src/main/java/io/syndesis/connector/jms/AbstractActiveMQConnector.java

```diff
@@ -39,6 +39,15 @@
     @Metadata(label = "basic", description = "Password for authorization credential")
     private String password;
 
+    @Metadata(label = "basic", description = "Client ID for durable subscriptions")
+    private String clientID;
+
+    @Metadata(label = "basic", description = "AMQ Broker X.509 PEM Certificate")
+    private String brokerCertificate;
+
+    @Metadata(label = "basic", description = "AMQ Client X.509 PEM Certificate")
+    private String clientCertificate;
```

Comment:
Needs to be added to `deployment.json` and Camel Connector JSON schema files

Reply:
I haven't added these since we are not exposing them at the moment, just allowing the MTIM exposed SSL connections. 
Otherwise, we'd need to ask the integration developer to extract keys from Broker truststore, convert them to PEM (for String *Certificate), and then paste them in Syndesis. Might be too much to ask according to @chirino. 
It's not difficult to add these fields, so maybe we can discuss that in another issue.

Reply:
@chirino , maybe we can add a couple of large text areas in Connection page for users to paste PEM certs, if they can obtain them. If they don't provide them, we can flash a big, flashing neon warning saying it's vulnerable to MTIM attacks. What do you think?